### DISCUSSION TITLE
商品编辑界面进货销售历程某些分辨率下关闭按钮显示位置不佳

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#34] [Q&A]
![image](https://github.com/densen2014/docs/assets/122285019/a835f499-9096-4b65-9c05-408f1ab6dddb)

**Comment by aaronameixa** [comment]
![image](https://github.com/densen2014/docs/assets/122285019/310c7b12-1dad-4452-939a-8528d076ff1d)
没看到

**Comment by densen2014** [comment]
什么版本? 我这里都是好的. 

![image](https://github.com/densen2014/docs/assets/8428709/0b38871a-6294-4b11-a6a4-a73d5912058a)

**Reply by aaronameixa** [reply]
2023-10-20

**Reply by densen2014** [reply]
开启新版编辑界面可以解决问题，因为你这个是旧版的编辑界面已经很少人用

**Reply by aaronameixa** [reply]
在哪里开启新版编辑界面？

**Reply by densen2014** [reply]
![image](https://github.com/densen2014/docs/assets/8428709/969809b1-d840-4628-8016-b7308645a10d)

**Comment by densen2014** [comment]
@aaronameixa 你是我们的代理吧

**Comment by densen2014** [comment]
好像已经是新版界面了, 有空截图看看你屏幕分辨率和放大率

**Comment by densen2014** [comment]
查询经营历程比较多的话, 管理者本机也可以试用新的整合界面功能

![image](https://github.com/densen2014/docs/assets/8428709/053cff2c-67d6-4424-a484-b79edfd4656e)

![image](https://github.com/densen2014/docs/assets/8428709/3ba1deef-df0c-437e-9b99-5d025dfa6dde)

![image](https://github.com/densen2014/docs/assets/8428709/558daee2-2e45-46a9-a41f-1c8f04b9b3dd)

**Comment by densen2014** [comment]
发布新版 7.11.22
